src/main/cpp/RobotContainer.cpp

```diff
@@ -1,67 +1,315 @@
-// Copyright (c) FIRST and other WPILib contributors.
-// Open Source Software; you can modify and/or share it under the terms of
-// the WPILib BSD license file in the root directory of this project.
+
+
+/****************************************** CREDITS: **********************************************************************************************************************
+
+		This code was written by Jacob Hagberg with a lot of help from Tyler Sieford.
+		Other members of the programming team who helped a lot were Austin and Eddie.
```

Comment:
This sort of thing would be great to put into the repository [README](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes).